docker-compose.yml

```diff
@@ -48,6 +48,35 @@ services:
       - 6001:6000
     depends_on:
       - yaba_api
+  
+  yaba_visualization:
+    build: ./visualization
+    networks:
+      - bety
+    volumes:
+      - './visualization:/code'
+      - '/code/node_modules'
+    ports:
+      - 80:8080
```

Comment:
What about 443 (https)? I'm not sure of the exact environment this is intended for, so this comment is to make sure it's supported

Reply:
TCP port 443 is the standard TCP port that is used for website which uses SSL. All secure transfers are done using the port as HTTPS connection encrypts traffic. However, HTTPS port 443 also supports sites to be available over HTTP connections.

Reply:
As long as https is supported.